application/src/main/java/org/togetherjava/tjbot/commands/AbstractCommand.java

```diff
@@ -1,7 +1,6 @@
-package org.togetherjava.tjbot.commands.example;
+package org.togetherjava.tjbot.commands;
```

Comment:
This change doesnt seem related to the tag system and will possibly clash with the changes introduced in #125 .

Reply:
No, it doesn't. The commits of that PR are also in this branch.

Reply:
Oh, I see. Then I should look at a diff between you and that branch instead of you and develop. But that also means that this PR has to be blocked until #125 is merged. Otherwise you will merge it together with your PR.

Reply:
#125 has already been merged.